climpred/classes.py

```diff
@@ -350,7 +351,30 @@ def _apply_xr_func(v, name, *args, **kwargs):
                 #            as ds[dim] and the dim doesn't exist as a key.
                 # DimensionError: This accounts for our custom error when applying
                 # some stats functions.
-                except (ValueError, KeyError, DimensionError):
+                except (ValueError, KeyError, DimensionError) as e:
+                    if args == tuple():
+                        func_name = None
+                    else:
+                        if callable(args[0]):
+                            func_name = args[0].__name__
+                        else:
+                            func_name = "not a callable"
```

Comment:
Can you add some comments here to explain what you are catching?

Reply:
thats the xr calls. now nicer message.